src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs

```diff
@@ -31,35 +42,49 @@ public bool IsChecked
             }
         }
 
+        /// <summary>
+        /// Callback that is called when the IsChecked parameter changed.
+        /// </summary>
         [Parameter] public EventCallback<bool> IsCheckedChanged { get; set; }
```

Comment:
this is not a correct comment for this parameter.
for now remove the documentation comment here

Reply:
Should I create a new issue for this one? so we won't forget it. Or would you give me a hint for a better description?

Reply:
this event is for the 2-way bound parameter which is related to the "@bind-" notation in blazor

Reply:
as I said, for now, there is no need for the documentation comment here.